slides/intro-02-data-budget.qmd

```diff
@@ -64,18 +64,6 @@ names(taxi)
 
 - Does this variable contribute to explainability?
```

Comment:
This (important) checklist is now relatively detached from the taxi data. I don't mind keeping it, just wanted to confirm.

Reply:
I still like the checklist, we can say something to the effect of "we went ahead and filtered the raw data set so we could focus on the modeling workflow in this workshop" or something to that effect